docs/source/backends.rst

```diff
@@ -74,6 +75,43 @@ torch.backends.cudnn
     and select the fastest.
 
 
+torch.backends.linalg
+^^^^^^^^^^^^^^^^^^^^^
+.. py:module::  torch.backends.linalg
+
+.. attribute::  torch.backends.linalg.preferred
+
+    .. warning:: This flag is experimental and subject to change.
+
+    A flag that lets pytorch prefer one of the many backend implementations when calling linear algebra functions on GPU.
+    Currently the available options are:
+
+    * `torch.linalg_default`
+    * `torch.linalg_cusolver`
+    * `torch.linalg_magma`
+
+    Usage:
+
+    * Use as a global flag, e.g. `torch.backends.linalg.preferred = torch.linalg_cusolver`
+    * Use the context manager, e.g. `with torch.backends.linalg.flags(preferred=torch.linalg_cusolver):`
+
+    Note: The usage of a backend is not guaranteed for all linear algebra operators even if that backend is set as preferred.
```

Comment:
alt. "When a library is preferred other libraries may still be used if the preferred library doesn't implement the operation(s) called."